diagnostic.md

```diff
@@ -4,6 +4,9 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
+$(".header header-logged-in true").find("").first()
+$(".container clearfix").find("").first()
```

Comment:
you dont need the .find() or the .first(), just the selectors which are what is between the `$('')`